packit_service/worker/jobs.py

```diff
@@ -561,6 +567,18 @@ def process_message(
             GithubAppInstallationHandler.get_signature(
                 event=event_object, job=None
             ).apply_async()
+        elif isinstance(
+            event_object, IssueCommentEvent
+        ) and self.is_fas_verification_comment(event_object.comment):
+            if GithubFasVerificationHandler(
+                package_config=None, job_config=None, event=event_object.get_dict()
+            ).pre_check():
+                event_object.comment_object.add_reaction(COMMENT_REACTION)
+                GithubFasVerificationHandler.get_signature(
+                    event=event_object, job=None
+                ).apply_async()
+            # should we comment about not processing if the comment is not
+            # on the issue created by us or not in packit/notifications?
```

Comment:
it is here because we can't process this as a "normal" jobs since there is no configuration with job which we can handle, so the behaviour is little bit like for GithubAppInstallationHandler above, but in that case if the event is an installation, we can directly tell to assign GithubAppInstallationHandler, in case of verification we need to check also the content of an issue comment, so it made sense to me to have it here and not try to initialise verification handler (which should be initialised only when we know that it is the handler we need for processing the event, since it ) if it can be just any other packit command comment/ any comment :thinking. Does it make sense?